src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+/**
+ * Main java class to create a new Duke object
```

Comment:
according to the module's coding standard, javadoc comments should end with a fullstop. This is different from git subject headers which states not to end with a fullstop.